packages/flame_fire_atlas/lib/flame_fire_atlas.dart

```diff
@@ -120,28 +168,43 @@ class AnimationSelection extends BaseSelection {
   }
 }
 
+/// FireAtlas is a mapping file that can hold several [Sprite]s and [SpriteAnimation]s.
+///
+/// Use [getSprite] and [getAnimation] to retrieve mapped assets.
 class FireAtlas {
+  /// Id of the FireAtlas.
```

Comment:
I think we could add a bit more detail on why they have ids, what it is used for, etc

Reply:
It is just an identifier used by the editor. Not sure what we could say here, any suggestion?

Reply:
Just mention it's an internal identifier, generated and used by the editor. just so users know they don't have to mess with it

Reply:
```suggestion
  /// Id of the FireAtlas, mainly used by the Fire Atlas Editor for file identification.
```